plugins/system/stats/stats.xml

```diff
@@ -34,9 +35,39 @@
 					size="30"
 					default="https://developer.joomla.org/stats/submit"
 					description="PLG_SYSTEM_STATS_URL_DESC"
-					label="PLG_SYSTEM_STATS_URL_LABEL"					
+					label="PLG_SYSTEM_STATS_URL_LABEL"
```

Comment:
I think we should make this hardcoded so that nobody can change it, it is too easy to misuse the plugin. (I kwon that is form the original but because we are changing it we can do this also)